src/engines/kotor/gui/main/main.cpp

```diff
@@ -24,6 +24,8 @@
 
 #include <boost/bind.hpp>
 
+#include <glm/gtc/matrix_transform.hpp>
```

Comment:
Please use `#include ""`, not `#include <>` for glm, because we're using our internal glm, not a system one.